chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -213,6 +240,11 @@ export const LiveChatWidgetStateful = (props: ILiveChatWidgetProps) => {
         if (state.appStates.conversationEndedByAgent) {
             endChat(props, chatSDK, setAdapter, setWebChatStyles, dispatch, adapter);
         }
+
+        //Listen to WidgetSize
+        BroadcastService.getMessageByEventName("WidgetSize").subscribe((msg: ICustomEvent) => {
```

Comment:
What is this used for?

Reply:
For restoring the widgetsize when maximizing chat.